astropy/modeling/core.py

```diff
@@ -336,17 +336,17 @@ def __call__(self, **kwargs):
 
             if param.default is param.empty:
                 raise ModelDefinitionError(
-                    'The bounding_box method for {0} is not correctly '
-                    'defined: If defined as a method all arguments to that '
-                    'method (besides self) must be keyword arguments with '
-                    'default values that can be used to compute a default '
-                    'bounding box.'.format(cls.name))
+                    f"The bounding_box method for {cls.name} is not correctly "
+                    "defined: If defined as a method all arguments to that "
+                    "method (besides self) must be keyword arguments with "
+                    "default values that can be used to compute a default "
+                    "bounding box.")
 
             kwargs.append((param.name, param.default))
 
         __call__.__signature__ = sig
 
-        return type(f'{cls.name}ModelBoundingBox', (ModelBoundingBox,),
+        return type(f"{cls.name}ModelBoundingBox", (ModelBoundingBox,),
```

Comment:
Why change single to double quote? I thought they are equivalent in Python.

Reply:
I prefer double quotes over single quotes when possible (its what the `black` formatter preferentially uses).